### PR TITLE
Add Travis CI badge

### PR DESCRIPTION
**Pull request by daniellevi1**
#7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # Open Pension Scraper
+
+[![Build Status][travis-image]][travis-url] [![Join the chat at https://gitter.im/open-pension/Lobby][gitter-image]][gitter-url]
```
Comment (nirgn975) [inline README.md]: You added both `travis` and `gitter`, keep only `travis`.

And this is just the urls for the image and the links. look at the [README](https://raw.githubusercontent.com/hasadna/open_pension/master/README.md) at the end of the file for the relevant links (they should be a little different, there are the links to open pension travis, we need our travis).

**Review by nirgn975 — CHANGES_REQUESTED** [review]

**Comment by nirgn975** [comment]
@daniellevi1 
By the way, you can go to the `code` section on this repo and move branch, to check how it will look:
selection_003

**Comment by daniellevi1** [comment]
@nirgn975  Thanks for the comments, Markdown language is used in other places than readme.me in github?

File: README.md [hunk 2]
```diff
@@ -16,3 +19,8 @@ Not yet.
 ## Contribute
 
 Just fork and do a pull request (;
+
+
+[travis-image]: https://api.travis-ci.org/nirgn975/open_pension_scraper.svg?branch=master
+[travis-url]: https://travis-ci.org/nirgn975/open_pension_scraper
+
```
Comment (nirgn975) [inline README.md:25]: You have 2 whitespaces at the the of the file, remove one.

**Review by nirgn975 — CHANGES_REQUESTED** [review]

**Comment by nirgn975** [comment]
@daniellevi1 
Yes, it's gaining popularity especially in websites who're appeal to developers, geeks, etc (see [Markdown](https://en.wikipedia.org/wiki/Markdown)).

**Comment by daniellevi1** [comment]
@nirgn975 Usually I leave 2 white spaces between 'paragraphs', it is common syntax to leave 1 in .md?

**Comment by nirgn975** [comment]
@daniellevi1 Yes, but not just in `md` files, in every file. Leave one space in the end of each and every file.

**Review by nirgn975 — APPROVED** [review]

**Pull request merged**